utils/commands.py

```diff
@@ -14,7 +14,7 @@
 import tarfile
 import time
 import yaml
-
+# diff 1
 from contextlib import contextmanager
```

Comment:
foo says:

[C0411(wrong-import-order), ] standard import "from contextlib import contextmanager" should be placed before "import yaml"
```